lua/flatten/guest.lua

```diff
@@ -73,14 +83,32 @@ M.init = function(host_pipe)
 	vim.api.nvim_create_autocmd("BufEnter", {
 		pattern = "*",
 		callback = function()
+			files = vim.tbl_map(function(bufnr)
+				local name = vim.fn.bufname(bufnr)
+				return name
+			end, vim.api.nvim_list_bufs())
+			nfiles = #files
+
 			if nfiles < 1 then
-				if require("flatten").config.nest_if_no_args == true then
+				local result = M.exec_on_host("return require'flatten'.config.callbacks.no_files()")
```

Comment:
Why does this need to be called on the host? If it's just a callback with no args, shouldn't it be callable in the same way from the guest without the overhead of an RPC request?

Reply:
All the other callbacks are called on the host right? Also I actually rely on it running on the host to focus that kitty window. If it were running on the guest then I would need a way to get the window id of the host, so it needs an rpc request anyway. 

I guess the callback could be made to run on the guest and the user can use the `exec_on_host` helper function.